contingency/contingency-api/src/main/java/com/powsybl/contingency/contingency/list/identifier/VoltageLevelAndOrderNetworkElementIdentifier.java

```diff
@@ -59,6 +59,16 @@ public Set<Identifiable<?>> filterIdentifiable(Network network) {
         }
     }
 
+    @Override
+    public Set<String> getNotFoundElements(Network network) {
+        VoltageLevel voltageLevel1 = network.getVoltageLevel(voltageLevelId1);
+        VoltageLevel voltageLevel2 = network.getVoltageLevel(voltageLevelId2);
+        if (voltageLevel1 == null || voltageLevel2 == null) {
```

Comment:
So if one of voltageLevelId1 or voltageLevelId2 is not found we get both in the list even the found one ?

Reply:
fixed. it will always return Collections.emptySet()